lib/fog/joyent/compute.rb

```diff
@@ -116,6 +118,10 @@ class Real
         def initialize(options = {})
           @connection_options = options[:connection_options] || {}
           @persistent = options[:persistent] || false
+          
+          if options[:joyent_ssl_verify_peer].to_s.downcase == 'false'
+            Excon.defaults[:ssl_verify_peer]=false
```

Comment:
This would "leak" out to other excon usage, which seems bad. I think if you merge it in to the connection options above instead it would properly scope this to just this case. In fact, you could probably just pass in `:connection_options => { :ssl_verify_peer => false }` and get it to work without code modification. Should be worth a try anyway, thanks!